src/lib/RuleResultRecombinator.ts

```diff
@@ -111,11 +123,15 @@ ${v.message.trim()}
 			column,
 			url
 		} = violation;
-		return `<testcase name="${fileName}">
-<failure message="${fileName}: ${line} ${htmlEscaper.escape(message.trim())}" type="${severity}">
-${severity}: ${message.trim()}
```

Comment:
This was failing because message wasn't being escaped a second time. Found it while scanning git://github.com/forcedotcom/almond.git

    The method 'createWorkObjects(List<Achievement_Assignment__c>,Map<Id,Achievement__c>)' has a cognitive complexity of 32.
    " type="3">
     3: 
     The method 'createWorkObjects(List<Achievement_Assignment__c>,Map<Id,Achievement__c>)' has a cognitive complexity of 32.